Exercise_3.go

```diff
@@ -0,0 +1,15 @@
+package main
+
+import "fmt"
+
+func Append(slice []int, index int, value int) {
```

Comment:
Please consider using a different name e.g. `SetAt` or change the implementation to actually append smth (-;